README.md

```diff
@@ -19,6 +19,10 @@ We currently have parser support for the following instruments:
 
 This code is published under the permissive MIT license because we believe that standardized instrument data is a benefit for everyone in science.
 
+## Python version
```

Comment:
could you move this to be with the `Install Python` section in the `Setup` section so it's all in one place?